packages/react/src/Bento/Bento.module.css

```
@@ -0,0 +1,1793 @@
+  .Bento__Item--xlarge-row-span-9 {
+    grid-row: auto/span 9;
+  }
+
+  .Bento__Item--xlarge-row-span-10 {
+    grid-row: auto/span 10;
+  }
+
+  .Bento__Item--xlarge-row-span-11 {
+    grid-row: auto/span 11;
+  }
+
+  .Bento__Item--xlarge-row-span-12 {
+    grid-row: auto/span 12;
+  }
+
+  /* XLARGE Column Start */
+  .Bento__Item--xlarge-column-start-1 {
+    grid-column-start: 1;
+  }
+
+  .Bento__Item--xlarge-column-start-2 {
+    grid-column-start: 2;
+  }
+
+  .Bento__Item--xlarge-column-start-3 {
+    grid-column-start: 3;
+  }
+
+  .Bento__Item--xlarge-column-start-4 {
+    grid-column-start: 4;
+  }
+
+  .Bento__Item--xlarge-column-start-5 {
+    grid-column-start: 5;
+  }
+
+  .Bento__Item--xlarge-column-start-6 {
+    grid-column-start: 6;
+  }
+
+  .Bento__Item--xlarge-column-start-7 {
+    grid-column-start: 7;
+  }
+
+  .Bento__Item--xlarge-column-start-8 {
+    grid-column-start: 8;
+  }
+
+  .Bento__Item--xlarge-column-start-9 {
+    grid-column-start: 9;
+  }
+
+  .Bento__Item--xlarge-column-start-10 {
+    grid-column-start: 10;
+  }
+
+  .Bento__Item--xlarge-column-start-11 {
+    grid-column-start: 11;
+  }
+
+  .Bento__Item--xlarge-column-start-12 {
+    grid-column-start: 12;
+  }
+
+  /* XLARGE Row Start */
+  .Bento__Item--xlarge-row-start-1 {
+    grid-row-start: 1;
+  }
+
+  .Bento__Item--xlarge-row-start-2 {
+    grid-row-start: 2;
+  }
+
+  .Bento__Item--xlarge-row-start-3 {
+    grid-row-start: 3;
+  }
+
+  .Bento__Item--xlarge-row-start-4 {
+    grid-row-start: 4;
+  }
+
+  .Bento__Item--xlarge-row-start-5 {
+    grid-row-start: 5;
+  }
+
+  .Bento__Item--xlarge-row-start-6 {
+    grid-row-start: 6;
+  }
+
+  .Bento__Item--xlarge-row-start-7 {
+    grid-row-start: 7;
+  }
+
+  .Bento__Item--xlarge-row-start-8 {
+    grid-row-start: 8;
+  }
+
+  .Bento__Item--xlarge-row-start-9 {
+    grid-row-start: 9;
+  }
+
+  .Bento__Item--xlarge-row-start-10 {
+    grid-row-start: 10;
+  }
+
+  .Bento__Item--xlarge-row-start-11 {
+    grid-row-start: 11;
+  }
+
+  .Bento__Item--xlarge-row-start-12 {
+    grid-row-start: 12;
+  }
+
+  /* XLARGE Flow */
+  .Bento__Item--xlarge-flow-column {
+    grid-auto-flow: column;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr;
+  }
+
+  .Bento__Item--xlarge-flow-row {
+    grid-auto-flow: row;
+    grid-template-rows: 1fr 1fr;
+    grid-template-columns: 1fr;
+  }
+
+  /* XLARGE Vertical Align */
+  .Bento__Item--xlarge-verticalAlign-start {
+    justify-content: start;
+  }
+
+  .Bento__Item--xlarge-verticalAlign-center {
+    justify-content: center;
+  }
+
+  .Bento__Item--xlarge-verticalAlign-end {
+    justify-content: end;
+  }
+
+  /* XLARGE Horizontal Align */
+  .Bento__Item--xlarge-horizontalAlign-start {
+    align-items: start;
+  }
+
+  .Bento__Item--xlarge-horizontalAlign-center {
+    align-items: center;
+  }
+
+  .Bento__Item--xlarge-horizontalAlign-end {
+    align-items: end;
+  }
+}
+
+/* XXLARGE breakpoint */
+@media screen and (min-width: 80rem) {
```

XXLARGE breakpoint value is `90rem`. 

```suggestion
@media screen and (min-width: 90rem) {
```